OneCommunityGlobal/HighestGoodNetworkApp
# change the order of formatting commands & run it

# Description
This PR is one of the re-enable Formatter and ESlint for our codes

## Related PRS (if any):
Jinchao did the previous works:
#938 
#983 

## Main changes explained:
My only manual change is in `package.json`
Other changes automated done by running command `npm run format`

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
~~3. log as admin user~~
~~4. go to dashboard→ Tasks→ task→…~~
~~5. verify function “A” (feel free to include screenshot here)~~
3. log as any user account
4. test if any function work as normal, which means nothing break because of this PR.

## Screenshots or videos of changes:

## Note:
FYI: we have a "makre Prettier and ESLint work together plan [here](https://docs.google.com/document/d/1RmQw3GPADLfRJ5eYnbFa7iMJA1chw98dF3qwef1omCQ/edit)"
